packages/@aws-cdk/aws-ecr/test/repository.test.ts

```diff
@@ -701,5 +701,21 @@ describe('repository', () => {
         repositoryName: 'a//a-a',
       })).toThrow(/must follow the specified pattern/);
     });
+
+    test('return value addToResourcePolicy', () => {
```

Comment:
Thanks for adding this test. Let's also add a test that looks at the actual output of calling this on a policy statement with concrete values. One test where the policy document is already defined and one where it is not.

Reply:
ok. Update to a test that checks all return values, not just singular